lib/node_modules/@stdlib/stats/base/dnanmskmax/include/stdlib/stats/base/dnanmskmax.h

```diff
@@ -19,7 +19,7 @@
 #ifndef STDLIB_STATS_BASE_DNANMSKMAX_H
 #define STDLIB_STATS_BASE_DNANMSKMAX_H
 
-#include <stdint.h>
+#include "stdlib/blas/base/shared.h"
```

Comment:
```suggestion
#include "stdlib/blas/base/shared.h"
#include <stdint.h>
```

The stdint header is necessary as the forward declarations use `uint8_t`.